frontend/src/scenes/onboarding/sdks/web-analytics/WebAnalyticsSDKInstructions.tsx

```diff
@@ -36,4 +37,5 @@ export const WebAnalyticsSDKInstructions: SDKInstructionsMap = {
     [SDKKey.WEBFLOW]: WebflowInstructions,
     [SDKKey.ANDROID]: AndroidInstructions,
     [SDKKey.REACT_NATIVE]: WebAnalyticsRNInstructions,
+    [SDKKey.FLUTTER]: FlutterInstructions,
```

Comment:
I am not sure why Mobile SDKs are in the web analytics instructions but since Flutter is the only missing, I added it

Reply:
(cc @robbie-c in case it's better we remove them)